demo/src/screens/MainScreen.js

```diff
@@ -199,13 +319,15 @@ class MainScreen extends Component {
           row
           spread
           paddingH-s5
-          paddingV-s2
-          onPress={() => this.openScreen(item)}
-          onLongPress={() => this.setDefaultScreen(item)}
+          paddingV-s4
+          onPress={this.openScreen}
+          customValue={item}
+          onLongPress={this.setDefaultScreen}
           activeBackgroundColor={Colors.$backgroundPrimaryHeavy}
```

Comment:
Let's change the `activeBackgroundColor` to something lighter, maybe `$backgroundPrimaryLight`